src/flows/shared/steps/OrganizeCollection/CollectionEditInfoForm.tsx

```diff
@@ -17,8 +17,8 @@ import { Collection } from 'types/Collection';
 type Props = {
   onNext: WizardContext['next'];
   collectionId: Collection['id'];
-  collectionTitle?: Collection['title'];
-  collectionDescription?: Collection['description'];
+  collectionTitle?: Collection['name'];
```

Comment:
maybe we can rename the prop names to match as well

Reply:
`collectionName`, `collectionNote` (or `collectionCollectorsNote` if we want to match explicitly)

Reply:
great call, done in a separate PR!